src/test/java/uk/gov/companieshouse/officerfiling/api/validation/OfficerAppointmentValidatorTest.java

```diff
@@ -1654,19 +1654,22 @@ void validateResidentialPostalCodeCharacters() {
         when(dto.getDateOfBirth()).thenReturn(LocalDate.of(1993, 1, 25));
         when(dto.getNationality1()).thenReturn("British");
         when(dto.getAppointedOn()).thenReturn(LocalDate.of(2023, 5, 14));
-        when(dto.getResidentialAddress()).thenReturn(AddressDto.builder(validResidentialAddress).postalCode("ゃ").build());
+        when(dto.getResidentialAddress()).thenReturn(AddressDto.builder(validResidentialAddress).country("England").postalCode("ゃ").build());
         when(dto.getServiceAddress()).thenReturn(validCorrespondenceAddressOutOfUK);
         when(dto.getConsentToAct()).thenReturn(true);
         when(apiEnumerations.getValidation(ValidationEnum.RESIDENTIAL_POSTAL_CODE_CHARACTERS)).thenReturn(
                 "Postal code must only include letters a to z, and common special characters such as hyphens, spaces and apostrophes");
+        when(apiEnumerations.getValidation(ValidationEnum.RESIDENTIAL_POSTCODE_UK_INVALID)).thenReturn(
+                "Enter a UK postcode. If the address is outside the UK, enter the address manually");
 
         final var apiErrors = officerAppointmentValidator.validate(request, dto, transaction,
                 PASSTHROUGH_HEADER);
         assertThat(apiErrors.getErrors())
                 .as("An error should be produced when Postal code contains illegal characters")
-                .hasSize(1)
+                .hasSize(2)
```

Comment:
I believe this was a confusing factor when implementing UK format validation. 
In the initial implementation, there was no order specified for the UK country - hence it was set as the last possible order of error, understandably these are not the errors we'd be displaying on the web anymore. 
However, I was under the impression that there's a 1-1 relation with the field <-> error message as there's an order or priority for the error messages. 

Changes look ok, from the development side, just unsure if we'd have another snag to fix.